src/cmd/tools/crane.go

```diff
@@ -88,3 +91,54 @@ func zarfCraneCatalog(cranePlatformOptions *[]crane.Option) *cobra.Command {
 
 	return craneCatalog
 }
+
+// Wrap the original crane list with a zarf specific version
+func zarfCraneList(cranePlatformOptions *[]crane.Option) *cobra.Command {
+	craneList := craneCmd.NewCmdList(cranePlatformOptions)
+
+	eg := `  # list the tags for a repo internal to Zarf
+  $ zarf tools registry ls internal/stefanprodan/podinfo
+
+  # list the tags for a repo hosted at reg.example.com
+  $ zarf tools registry ls reg.example.com/stefanprodan/podinfo`
+
+	craneList.Example = eg
+	craneList.Args = nil
+
+	originalListFn := craneList.RunE
+
+	craneList.RunE = func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			message.Fatal(nil, lang.CmdToolsCraneListNoRepoSpecified)
+		}
+
+		if !strings.HasPrefix(args[0], "internal/") {
```

Comment:
Instead of having an `internal/` prefix, we should have the user provide the full host and check if the host matches the value in the `zarfState.RegistryInfo.URL` secret that gets loaded later in this function.

If the registry hostname matches, we can use the username/password from the state to configure the `authOption`. Otherwise we will not provide an  `authOption` and let crane use the values it finds on the hosts local credential store to make the request.


I.e.
```bash
zarf init --registry-url="my-registry.com"

zarf tools crane registry ls myregistry/myrepo/myimage      # use username/password from secret in state
```